service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/C2UploadedEventHandlerTest.java

```diff
@@ -205,7 +206,7 @@ void shouldNotNotifyAllocatedJudgeOnC2UploadWhenAllocatedJudgeDoesNotExist() {
             c2UploadedEventHandler.sendC2UploadedNotificationToAllocatedJudge(
                 new C2UploadedEvent(caseData, c2DocumentBundle));
 
-            verify(notificationService, never()).sendEmail(any(), any(), anyMap(), any());
+            verify(notificationService, never()).sendEmail(any(), any(String.class), anyMap(), any());
```

Comment:
if you have overloaded methods, this check in no longer string enough, it can happen that code uses method that accepts Collection of emails and not single one, ant this test wont spot it. Use verifyNoInteractions(notificationService); instead